packages/devextreme/js/__internal/ui/list/m_list.edit.decorator.selection.ts

```diff
@@ -74,13 +74,26 @@ registerDecorator(
       this.callBase.apply(this, arguments);
 
       const { $itemElement } = config;
+
+      this._updateItemAria($itemElement);
+
       const control = this._controlWidget.getInstance($itemElement.find(`.${this._controlClass}`));
 
       eventsEngine.on($itemElement, 'stateChanged', (e, state) => {
         control.option('value', state);
       });
     },
 
+    _updateItemAria($itemElement) {
+      const text = $itemElement.text();
```

Comment:
I'm a bit worried about getting ` $itemElement.text()` - how will it work with templates when content is huge?
I see in treeView it's implemented a bit another way : `label: this._displayGetter(node.internalFields.item) || '',.
Also please check is this really the best place to apply aria attributes

Reply:
[c19e53a751cd9eb55c8fa738214d2017f4c5b39c, c3eb095cdae339944cf3b7594bc501cb64c0185a]